elastic/integrations
# [kibana] align mappings with metricbeat

### Summary

**This change should be tested with https://github.com/elastic/kibana/pull/138224**

Closes https://github.com/elastic/integrations/issues/3915 
Closes https://github.com/elastic/kibana/issues/137691

Align the kibana data streams' mappings with the [metricbeat ones](https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/core/src/main/resources/monitoring-kibana-mb.json) and add data streams to support the Rules metricsets introduced in https://github.com/elastic/beats/pull/29899

### Testing
When testing we have to install the elasticsearch package as well otherwise the kibana section in Stack Monitoring won't show up.

- update the kibana fleet configuration to preinstall the elasticsearch and kibana package:
  - create a new stack profile with `elastic-package profiles create stack_monitoring-kibana --from default`
  - edit the fleet configuration of the profile at `~/.elastic-package/profiles/stack_monitoring-kibana/stack/kibana.config.8x.yml` with [this one](https://gist.github.com/klacabane/2da09edc1d3ff84d6cf7a04c3de07993#file-kibana-config-8x-yml-L23-L87)
- build the elasticsearch and kibana packages with `elastic-package build`
- start a 8.5.0 stack within integrations repo `elastic-package stack up -v -d --profile stack_monitoring-kibana --version 8.5.0-SNAPSHOT`
- verify all kibana data streams are populated (`stats` `status` `cluster_actions` `cluster_rules` `node_actions` `node_rules`)
- inspect Stack Monitoring UI with a local kibana running https://github.com/elastic/kibana/pull/138224

TODO:
- update README
